Source/ResponseObjects.swift

```diff
@@ -80,7 +80,9 @@ public final class S3BucketObjectList: ResponseObjectSerializable {
             let dateString = xml["LastModified"].element?.text,
             let date =  dateFromS3Date(dateString),
             let sizeString = xml["Size"].element?.text,
-            let size = Float(sizeString) else { return nil }
+            let size = Int64(sizeString) else { return nil }
+
+		print("sizeString='\(sizeString)' size=\(size) path=\(path)")
```

Comment:
Please remove unneeded logging from the PR and I'd be happy to accept it. Thank you!